apps/api/src/helper/object.helper.ts

```diff
@@ -3,7 +3,7 @@ import { cloneDeep, isArray, isObject } from 'lodash';
 
 export function hasNotDefinedValuesInObject(aObject: Object): boolean {
   for (const key in aObject) {
-    if (aObject[key] === null || aObject[key] === null) {
+    if (aObject[key] === null || aObject[key] === undefined) {
```

Comment:
Wow, this looked strange. Thanks for creating a patch @virus231. How did you notice that?

Reply:
Just looking at your code